### ISSUE TITLE
Community Spaces have momentum with a clear view on metrics to define success and ways to effectively interact with developers

### ISSUE THREAD

**Issue by brianking** [mozilla/participation-org#424]
## Objective Goal:

Our 3 community spaces continue with momentum running events focused on developers, with a clear view on metrics to define success and ways to effectively interact with developers

R: Space Stewards
A: @brianking
C: @jmswisher 
I: @geroter 
## Tasks & To Dos Related to This Project:
- https://github.com/mozilla/community-spaces
## Sub-Goal Success:
- [ ] Build a metrics dashboard; Mentor call for 2 more events types (R: @brianking @jmswisher @couci Space Stewards A: @brianking C: I: )
- [ ] Write a proposal for a common way/tool for all spaces to handle registration and follow-up surveys at spaces (R: Space Stewards @brianking A: @brianking C: I: )
- [x] A design sprint is setup to run in London for developer engagement tools and programs (R: Natalia @jmswisher @brianking A: Natalia C: Ali Spivek @geroter I: )
- [ ] Blog post on momentum of the dev events program, and the Space program in general (R: @brianking A: @brianking C: I: ) 
## Related Issues & Links:
- https://wiki.mozilla.org/Marketing/Developer/Meetups_HOWTO

**Comment by brianking** [comment]
tl;dr Made some slow progress on this during this heartbeat, but nothing to completion. Spaces are only planning a month ahead at most at this stage.

After London, @couci will be make a dashboard based on numbers we have:

https://docs.google.com/a/mozilla.com/spreadsheets/d/1h8nsKT00FVrRLUu1-2hpM-p0GD3eHztYB5AqdKm0suw/edit?usp=sharing

Than I'll be writing a blog post about it.

We have a session in London and @hmitsch is helping facilitate:

http://mozillalondonallhands2016.sched.org/event/72oq/community-spaces-meeting

**Comment by brianking** [comment]
Moved now to #435
